### PR TITLE
[IMP] calendar, repair: move action from root menu to new menu

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, the repair orders is directly
linked with the root menu of repair application and if you user opens the repair app, repair orders
action is shown as it is linked with the main menu.

similarly in the calendar app, the calendar is linked
with the root menu calendar.

but now if user move to reporting or configuration menu of repair app, there is
 no menu to access the repair orders.

with the new milk redesign on clicking the root menu user will be taken to home screen, where us
there is hamburger menu in previous design to navigate to home screen.

so only way for user to access the repair order menu is to go back to home screen and come back.

similarly in the calendar app, if user takes the
configuration menu, in order to access the calendar, user
has to go to home screen and come back.

after this commit, a new sub menu is introduced in both
app and moved the action linked with root menu to
the newly added sub menu.

Issue:
https://github.com/odoo/odoo/assets/27989791/c21945d0-84f2-4037-a15e-b4d555590dfc

After this commit:
Screenshot from 2023-06-24 12-06-00

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by niyasraphy** [comment]
hi @Whenrow ,
if you have time, can you review this.
thanks

**Comment by niyasraphy** [comment]
hi @arj-odoo ,
if you have time, can you review this.
thanks

**Comment by arj-odoo** [comment]
> hi @arj-odoo , if you have time, can you review this. thanks

It looks good. Could you rebase and ping me when it's green? I will launch an enterprise build to make sure nothing is broken (I am not sure if you can do it without creating an empty branch) and I will review on behalf the calendar team.

**Comment by niyasraphy** [comment]
hi @arj-odoo rebased and runbot is green now

**Comment by niyasraphy** [comment]
hi @Whenrow ,
if you have time, can you review this PR
thanks

File: addons/calendar/views/calendar_views.xml [hunk 1]
```diff
@@ -470,10 +470,17 @@
         id="mail_menu_calendar"
         name="Calendar"
         sequence="10"
-        action="action_calendar_event"
         web_icon="calendar,static/description/icon.png"
         groups="base.group_user"/>
 
+    <menuitem
+        id="menu_mail_calendar"
```
[the remark this answers is not in the document]
Comment (niyasraphy) [reply addons/calendar/views/calendar_views.xml]: thanks for the feedback @arj-odoo ,
updated the PR

**Review by arj-odoo — APPROVED** [review]

**Comment by niyasraphy** [comment]
hi @Whenrow ,
if you have time, can you review this PR, inventory/repair side is pending.

thanks

**Comment by Whenrow** [comment]
@niyasraphy Thank you 
robodoo r+

**Comment by niyasraphy** [comment]
hi @Whenrow ,
seems robodoo missed this ..

**Comment by arj-odoo** [comment]
@robodoo r+

**Comment by niyasraphy** [comment]
@arj-odoo ,
should i need to rebase or ...

**Comment by arj-odoo** [comment]
> @arj-odoo , should i need to rebase or ...

You can rebase. Maybe part of the issue is that the branch name don't follow our guidelines.
next time, could you use a name like master-repair_menu-niyasraphy It would be more easy to follow.

**Comment by niyasraphy** [comment]
sure @arj-odoo will follow the guidelines for the branch naming from next pr.

for this pr i rebased now, should we do anything further to get this merged or ...

**Comment by niyasraphy** [comment]
@arj-odoo rebased

**Comment by Whenrow** [comment]
robodoo r+

**Pull request closed without merging**